# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/unpaper/hotfix-sphinx.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- unpaper-7.0.0/meson.build.vanilla	2025-07-04 16:57:44.837239846 +0200
+++ unpaper-7.0.0/meson.build	2025-07-04 16:58:42.582387761 +0200
@@ -36,8 +36,9 @@
     install : true,
 )
 
-sphinx = find_program('sphinx-build', required: true, version: '>= 3.4')
+sphinx = find_program('sphinx-build', required: false, version: '>= 3.4')
 
+if sphinx.found()
 custom_target(
     'man',
     command: [sphinx, '-b', 'man', join_paths(meson.source_root(), 'doc'), '@OUTDIR@'],
@@ -46,6 +47,7 @@
     install: true,
     install_dir: join_paths(get_option('prefix'), get_option('mandir'), 'man1'),
 )
+endif
 
 pymod = import('python')
 python = pymod.find_installation(required: false, modules: ['pytest', 'PIL'], disabler: true)
